AUTHORS.rst

```diff
@@ -3,38 +3,36 @@
 People
 ------
 
-This work is made available by a community of people, amongst which
+This work is made available by a community of people, which
+originated from
 the `INRIA Parietal Project Team <https://team.inria.fr/parietal/>`_
-and the `scikit-learn <http://scikit-learn.org/>`_ folks, in
-particular:
-
-* Alexandre Abraham
-* `Alexandre Gramfort <http://alexandre.gramfort.net>`_
-* Vincent Michel
-* Bertrand Thirion
-* `Fabian Pedregosa <http://fa.bianp.net/>`_
-* `Gael Varoquaux <http://gael-varoquaux.info/>`_
-* Philippe Gervais
-* Michael Eickenberg
-* Danilo Bzdok
-* Loïc Estève
-* Kamalakar Reddy Daddy
-* Elvis Dohmatob
-* Alexandre Abadie
-* Andres Hoyos Idrobo
-* Salma Bougacha
-* Mehdi Rahim
-* Sylvain Lanuzel
-* `Kshitij Chawla <https://github.com/kchawla-pi>`_
-
-Many of also contributed outside of Parietal, notably:
-
-* `Chris Filo Gorgolewski <http://multiplecomparisons.blogspot.fr/>`_
-* `Ben Cipollini <http://cseweb.ucsd.edu/~bcipolli/>`_
-* Julia Huntenburg
-* Martin Perez-Guevara
-
-Thanks to M. Hanke and Y. Halchenko for data and packaging.
+and the `scikit-learn <http://scikit-learn.org/>`_ but grew much further.
+
+An up-to-date list of contributors can be seen in on `gitub
+<https://github.com/nilearn/nilearn/graphs/contributors>`_
+
+Additional credit goes to M. Hanke and Y. Halchenko for data and packaging.
+
+.. _core_devs:
+
+Core developers
+.................
+
+The nilearn core developers are:
+
+* Alexandre Gramfort https://github.com/agramfort
+* Ben Cipollini https://github.com/bcipolli
```

Comment:
See, you do this during Hacktoberfest, and now I feel like it's a good opportunity to make some new contributions :) Will check in soon!

Reply:
This was mere luck.

But we are hoping to release soonish. The new HTML reports are totally worth a release!

And we'd be so happy to welcome more of your contributions!